tests/all/memory_creator.rs

```diff
@@ -108,7 +108,14 @@ mod not_for_windows {
     }
 
     unsafe impl MemoryCreator for CustomMemoryCreator {
-        fn new_memory(&self, ty: MemoryType) -> Result<Box<dyn LinearMemory>, String> {
+        fn new_memory(
+            &self,
+            ty: MemoryType,
+            reserved_size: Option<u64>,
+            guard_size: u64,
+        ) -> Result<Box<dyn LinearMemory>, String> {
+            assert_eq!(guard_size, 0);
```

Comment:
Actually that example is allocating one guard page

Reply:
True yeah, I was mostly just getting this test to pass. Since the JIT is configured that there might not be a guard page then the guard page allocation here shouldn't be necessary.